addons/website_slides/models/slide_slide.py

```diff
@@ -1285,6 +1285,32 @@ def _default_website_meta(self):
     # Data / Misc
     # ---------------------------------------------------------
 
+    def is_category_completed(self):
+        """ Determine if this slide category is completed.
```

Comment:
same here, it can be private (always do private method, and turn it public if you need to)